src/genkey/clu_genkey.c

```diff
@@ -799,6 +799,209 @@ int wolfCLU_genKey_RSA(WC_RNG* rng, char* fName, int directive, int fmt, int
 #endif
 }
 
+
+int wolfCLU_genKey_Dilithium(WC_RNG* rng, char* fName, int directive, int fmt,
+                            int keySz, int level, int withAlg, int keyType)
+{
+#ifdef HAVE_DILITHIUM
+    dilithium_key key;
+    XFILE  file;
+    int    ret = WOLFCLU_SUCCESS;
+
+    int   fNameSz;
+    int   fExtSz      = 6;
```

Comment:
Add a comment on why 6 is the value chosen here.

Reply:
This value is sizeof of file extension such as ".priv\0" and ".pub\0\0".

Reply:
fixed it